File: Unidad3/Java/Programa13.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa13;

import java.util.Scanner;

/**
 *
 * @author LaboratorioLTI2
 */
public class Programa13 {

    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int c, total, num, negativos = 0, positivos=0;
        System.out.println("Escribe el total de numeros a procesar!");
        total=sc.nextInt ();
        
        for (c=1; c<=total; c++){
        System.out.println("Escribe tu numero: ");
        num=sc.nextInt();
        if (num>0) {
            positivos++;}
        else if(num<0){
          negativos++;}
                System.out.println("Cantidad de numeros Positivos="+positivos);
                System.out.println("Cantidad de numeros Negativos="+negativos);
    }
}

}

File: Unidad4/Java/Programa13.js
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa13u4;

/**
 *
 * @author LaboratorioLTI2
 */
import java.util.Scanner;
public class Programa13u4 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        float [][] temp = new float [5][7];
        for(int f=0; f<5; f++){
           for(int c=0; c<7; c++){
        
        
                System.out.println("Escribre la Temperatura en" +f+ ","+c+")");
                        temp [f][c] = sc.nextFloat();
        }

    }
        float[][] tempF = new float [5] [7];
        for(int f=0; f<5; f++){
           for(int c=0; c<7; c++){
           tempF [f][c] = (temp[f][c] * 9/5) + 32;
           System.out.println("F en ("+f+","+c+")"+tempF[f][c]);
           
           }
                   System.out.println();
        }
}
}